workout-app/src/components/weekpot.tsx

```diff
@@ -1,11 +1,71 @@
-import React from 'react'
+import React, { useState } from 'react'
+import "@fontsource/reenie-beanie"
+
+const weekpot = ({toggleBoard, handleToggle}) => {
+
+  const nameMapping = {
+    1: 'Sandeep Suri',
+    2: 'Harvir Sidhu',
+    3: 'Jimi Patel',
+    4: 'Deshane Medley',
+    5: 'Shanthosh Yogendren',
+    6: 'Gurkirat Gill'
```

Comment:
Great Work Sunny. I'm assuming once we generate the intake form you will introduce logic to make this dynamic right?

Reply:
Hey both @jimip85 @sidhuharvir I looked into your requests and applied those changes. Please take a look at the new changes and let me know if it's good to merge!